sklearn/impute/_iterative.py

```diff
@@ -544,20 +545,23 @@ def _initial_imputation(self, X, in_fit=False):
 
     @staticmethod
     def _validate_limit(limit, limit_type, n_features):
-        """Validate the limits (min/max) of the feature values
-        Converts scalar min/max limits to vectors of shape (n_features,)
+        """Validate the limits (min/max) of the feature values.
+
+        Converts scalar min/max limits to vectors of shape `(n_features,)`.
 
         Parameters
         ----------
         limit: scalar or array-like
-            The user-specified limit (i.e, min_value or max_value)
-        limit_type: string, "max" or "min"
-            n_features: Number of features in the dataset
+            The user-specified limit (i.e, min_value or max_value).
+        limit_type: str, "max" or "min"
```

Comment:
```suggestion
        limit_type : {"min", "max"}
```

Reply:
I didn't do it like this in the beginning because the code only controls if the `limit_type == 'max'`, any other string would give you the other option.
The code used is: `limit_bound = np.inf if limit_type == "max" else -np.inf`.
Nevertheless, these are the only two options used in the code, so it would be fine using that in the docstring.